.github/dependabot.yml

```diff
@@ -38,3 +38,19 @@ updates:
       # Skip github.com/elastic/mito because it requires documentation updates.
       - dependency-name: github.com/elastic/mito
     open-pull-requests-limit: 2
+
+  # GitHub actions
+  - package-ecosystem: "github-actions"
```

Comment:
Let's change the code owner for the `.github` directory to `@elastic/ingest-eng-prod` as part of this change so the right people see these PRs.

https://github.com/elastic/beats/blob/d23b4d33622137f7245b628d77d15f8ecfef3a0d/.github/CODEOWNERS#L22

Reply:
See https://github.com/elastic/beats/pull/38184/commits/580998274d152230e540017345a2f3e5bfba3460